apps/bors_frontend/web/command.ex

```diff
@@ -1,3 +1,18 @@
+defmodule BorsNG.Command.MultiParse do
+  defmacro multi_parse(phrases, param, [do: result]) do
```

Comment:
It's a macro because it's generating other functions. Thinking further about it, that's probably a dumb way to go about this.